api/src/main/resources/workbench.yaml

```diff
@@ -2027,12 +2027,13 @@ paths:
     parameters:
       - $ref: '#/parameters/workspaceNamespace'
       - $ref: '#/parameters/workspaceId'
-    get:
+    post:
```

Comment:
I have an explanation in my PR description. 

- Swagger 2 does not support objects in GET methods since they do not support objects in path and query parameters. Swagger 3 does

- I changed the getDataSetsByResourceId to be a POST request so I can send a body and parameterize it as an object through Swagger. We can change this back to a GET when we upgrade to Swagger 3.

Reply:
Ah, my bad, thank you.